esmvalcore/experimental/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""ESMValCore experimental API module."""
+
+from ._warnings import warnings
+
+warnings.warn(
+    '\n  Thank you for trying out the new ESMValCore API.'
+    '\n  Note that this API is experimental and may be subject to change.'
+    '\n  More info: https://github.com/ESMValGroup/ESMValCore/issues/498', )
+
+from .config import CFG  # noqa: E402
```

Comment:
```suggestion
from .config import CFG
```
is `noqa` needed here? I think flake8 recognizes that `CFG` is used in `__all__`

Reply:
E402 is about imports not being at the top of the file. I want to have the warning as the very first thing on import of `esmvalcore.experimental`, so it must come first. This is why I added the noqa.